examples/postgresql-ssl/postgresql.pp

```diff
@@ -0,0 +1,53 @@
+$password = 'sensu'
+
+class { 'sensu::agent':
+  backends => ['sensu-backend:8081'],
+}
+class { 'postgresql::globals':
+  manage_package_repo => true,
+  version             => '9.6',
+}
+class { 'postgresql::server':
+  listen_addresses => '*',
+}
+file { 'postgresql_ssl_key_file':
+  ensure => 'file',
+  path   => "${postgresql::server::datadir}/${trusted['certname']}.pem",
+  source => "/etc/puppetlabs/puppet/ssl/private_keys/${trusted['certname']}.pem",
+  owner  => 'postgres',
+  group  => 'postgres',
+  mode   => '0600',
+}
+postgresql::server::db { 'sensu':
+  user     => 'sensu',
+  password => postgresql_password('sensu', $password),
+}
+postgresql::server::pg_hba_rule { 'allow access to sensu database':
```

Comment:
add a new line between resources so it is easier to read

Reply:
Added newlines